stable/pomerium/OWNERS

```diff
@@ -1,5 +1,7 @@
 approvers:
 - desimone
+- travisgroth
 reviewers:
 - desimone
-- victornoel
\ No newline at end of file
+- victornoel
+- travisgroth
```

Comment:
missing a newline?
and please update the maintainers section in Chart.yaml :)

Reply:
Done 👍